deploy/clusterrole.yaml

```diff
@@ -0,0 +1,29 @@
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: acme-controller
+  annotations:
+    openshift.io/reconcile-protect: "true"
```

Comment:
I don't think we need to protect it against reconciliation as there is no such role currently in OpenShift, right?